datasets/dining_prefs.csv

```diff
@@ -0,0 +1,52 @@
+Ada Cora 1

```

Comment:
I thought that dataset was being saved in the cloud and accessed via:  https://data.rapids.ai/cugraph/datasets/dining_prefs.csv

so why do we needed that datafile here?